services/provider/server/server.go

```diff
@@ -171,7 +171,13 @@ func (s *OCSProviderServer) OnboardConsumer(ctx context.Context, req *pb.Onboard
 		storageConsumerUUID = string(storageConsumer.UID)
 	}
 
-	return &pb.OnboardConsumerResponse{StorageConsumerUUID: storageConsumerUUID}, nil
+	enforceHostNWCSI := false
+	addressRanges := rookCephv1.AddressRangesSpec{}
+	if storageCluster.Spec.HostNetwork && storageCluster.Spec.Network.AddressRanges != &addressRanges {
```

Comment:
why do you need to compare pointers here?

Reply:
changed